ErsatzTV/client-app/src/views/FFmpegProfilesPage.vue

```diff
@@ -27,10 +40,67 @@ export default class FFmpegProfiles extends Vue {
                 value: 'resolution'
             },
             { text: this.$t('ffmpeg-profiles.table.video'), value: 'video' },
-            { text: this.$t('ffmpeg-profiles.table.audio'), value: 'audio' }
+            { text: this.$t('ffmpeg-profiles.table.audio'), value: 'audio' },
+            { text: 'Actions', value: 'actions', sortable: false }
         ];
     }
 
+    addRecord() {
+        this.$router.push({
+            name: 'add-ffmpeg-profile'
+        });
+    }
+
+    deleteRecord(record: any, recordName: any) {
+        this.$swal
+            .fire({
+                title: 'Are you sure?',
+                text: 'Delete "' + recordName + '" FFmpeg Profile?',
+                icon: 'warning',
+                iconColor: '#4CAF50',
+                showCancelButton: true,
+                confirmButtonText: 'Yes'
```

Comment:
If we have `Yes` as an option, the cancel button should be `No`.

Reply:
Done!